templates/registration/includes/profile_stats.html

```diff
@@ -0,0 +1,56 @@
+{% load staticfiles %}
+{% load humanize %}
+
+<div class="investments">
+    <h2>EARNINGS</h2>
+    <div class="break">
+        <hr>
+    </div>
+</div>
+<h1 id="percentage"></h1>
+<h4 style="color: #4cbd7e;""><i class="fa fa-arrow-up" aria-hidden="true"></i>${{ total_earned|floatformat:2|intcomma }} | <span style="color: #4d96ad">${{ total_investments|floatformat:2|intcomma }}</span></h4>
```

Comment:
You have an extra quote on this line that messes everything after it up. Pay close attention to the syntax colouring.